core/src/main/java/org/opensearch/sql/planner/physical/datasource/DataSourceTableScan.java

```diff
@@ -45,7 +45,7 @@ public String explain() {
   @Override
   public void open() {
     List<ExprValue> exprValues = new ArrayList<>();
-    Set<DataSourceMetadata> dataSourceMetadataSet = dataSourceService.getDataSourceMetadata(true);
+    Set<DataSourceMetadata> dataSourceMetadataSet = dataSourceService.getDataSourceMetadata(false);
```

Comment:
why change to false?

Reply:
This change is to remove @opensearch default datasource from output of `show datasources`. Since UI cannot show/Vamsi said that UI should not show this value